### PR TITLE
Update tasks to pull from image mirror

### PR DESCRIPTION
**Pull request by apburnes**
## Changes proposed in this pull request:
- [x] Update tasks to point to internal registry image mirror
- [ ] Add pipenv image to interal registry

## security considerations
Can pull images without rate limiting

### CODE REVIEW

File: ci/aws-rds-storage.yml [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ---
 platform: linux
 image_resource:
-  type: docker-image
+  type: registry-image
   source:
     repository: kennethreitz/pipenv
```
Comment (apburnes) [inline ci/aws-rds-storage.yml:6]: Can we push this to the mirror from the jumpbox?

Comment (bengerman13) [reply ci/aws-rds-storage.yml:6]: not sure what you mean - are you asking if this image can be mirrored through our registry mirror?

**Review by bengerman13 — APPROVED** [review]

File: ci/aws-rds-storage.yml (hunk 1 above)
Comment (apburnes) [reply ci/aws-rds-storage.yml:6]: Do we have to push `kennethreitz/pipenv` first to the mirrored registry to be able to pull it right away?

Comment (bengerman13) [reply ci/aws-rds-storage.yml:6]: no - the mirror is a pull-through cache mirror, so if we try to pull an image and it doesn't exist, the mirror pull it from docker hub and store it locally, then serve the local copy to us.

Comment (apburnes) [reply ci/aws-rds-storage.yml:6]: Copy that.  👍

**Pull request merged**